SWLOR.Game.Server/Service/SkillService/SkillCategoryType.cs

```diff
@@ -16,6 +16,13 @@ public enum SkillCategoryType
         Languages = 4,
     }
 
+    public enum CombatPointCategory
```

Comment:
Let's put this enum in its own CombatPointCategory.cs file in the SkillService folder. The only time I double up is when the enum is directly related to the attribute. A CombatPointCategory is a property of the attribute and could have value elsewhere so it should live on its own.

Reply:
Also, please suffix the enum with "Type" so it's CombatPointCategoryType. The code base is slightly inconsistent with this - mostly in the Core folder - but it helps a lot with differentiating between enums and attributes.